source/documentation/standards/metrics.html.md.erb

```diff
@@ -0,0 +1,59 @@
+---
+title: Metrics
+last_reviewed_on: 2021-03-15
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## User needs
+
+Users should be able to label metric data so that it is compatible with
+UK Government PaaS Exporter metrics and common across applications.
+
+```
+<metric name>{<label name>=<label value>, ...}
+```
+  
+## Principles
+
+- **HELP**  Description of the Metric
+- **TYPE**  Should be one of the [Prometheus Metric Types](https://prometheus.io/docs/concepts/metric_types/)
+- **Metric Name** The name of the metric should identify what it contains. 
+- **Label Name** labels are used to differentiate the characteristics of the thing that is being measured
+  
+Where possible use the same metric name and label name across different applications. However, ensure that the value is consistent in scale and the data it
+represents.
+
+```
+cpu_usage_percentage{ app="get-into-teaching-api" }  75 
```

Comment:
This looks like a metric provided by paas. Could we list a more custom one for the example?